debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -241,6 +263,19 @@ public void dispatchSchemaChangeEvent(T dataCollectionId, SchemaChangeEventEmitt
         schemaChangeEventEmitter.emitSchemaChangeEvent(new SchemaChangeEventReceiver());
     }
 
+    public void dispatchSchemaChangeEvent(List<T> dataCollectionIds, SchemaChangeEventEmitter schemaChangeEventEmitter) throws InterruptedException {
+        final boolean anyNonfilteredEvent = dataCollectionIds.stream()
```

Comment:
Let's avoid stream ops in this inner loop method.

Reply:
Done